ubuntudesign/www.ubuntu.com
# Cloud form cleanup

## Done

* moved cloud forms to use new opt-in fields
* got cloud forms to use a single new form
* got client forms to use a single new form
* redirected all old contact-us forms to new ones
* added a get parameter for thank-you pages to mimic the contact-us logic
* remove dead tender forms

## QA

1. go to all /cloud contact forms and see that it goes to the new forms
2. go to /desktop/contact-us and see that still works
3. go to /tablet/contact-us and see that it still works
4. go to /internet-of-things contact us form and see that it still works
5. go to /server/contact-us and see that it still works

## Issue / Card

[trello card](https://trello.com/c/lOUBdXoy)
